### PR TITLE
Features/support ar 7 2

### PR DESCRIPTION
**Pull request by flyerhzm**
It supports activerecord 7.2

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -5,22 +5,80 @@ on:
...
+  test_rails_7_2:
+    name: Ruby 3.3, Rails 7.2
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        ruby: ['2.7', '3.0']
-        rails: ['5.2', '6.0', '6.1', '7.0', '7.1']
-        exclude:
-        - ruby: '3.0'
-          rails: '5.2'
```
Comment (grosser) [inline .github/workflows/actions.yml:17]: if you don't want to deal with bad combinations then do:
```
include:
  - rails: 2.7
     ruby: 3.0
```
or something like that, but I'd prefer that over copy-paste solutions

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .bundle
 .ruby-version
 /gemfiles/.bundle
+/gemfiles/*.gemfile.lock
```
Comment (grosser) [inline .gitignore:4]: I'm leaving these on purpose so we have a reproducible good setup

File: ar_after_transaction.gemspec [hunk 3]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new 'ar_after_transaction', ARAfterTransaction::VERSION do |s
   s.homepage = 'http://github.com/grosser/ar_after_transaction'
   s.files = `git ls-files lib Readme.md`.split("\n")
   s.required_ruby_version = '>= 2.6.0'
-  s.add_runtime_dependency 'activerecord', '>= 5.2.0', '< 7.2'
+  s.add_runtime_dependency 'activerecord', '>= 5.2.0', '<= 7.2'
```
Comment (grosser) [inline ar_after_transaction.gemspec:12]: we also want to support 7.2.1 etc
```suggestion
  s.add_runtime_dependency 'activerecord', '>= 5.2.0', '< 7.3'
```

File: .github/workflows/actions.yml (hunk 1 above)
Comment (grosser) [reply .github/workflows/actions.yml:17]: can drop 2.7/3.0 and 5.2 + 6.0 if that makes things easier

**Comment by grosser** [comment]
needs to be added to CI ?

**Comment by flyerhzm** [comment]
@grosser yes, please give CI a try.

**Comment by grosser** [comment]
"You are trying to install in deployment mode after changing"

`rake bundle_all` should help
... maybe first remove ruby 2.7 and 3.0 to simplify the setup
can also kick out rails 6.0

**Comment by flyerhzm** [comment]
@grosser I have updated `.gemfile.lock`s.

**Comment by grosser** [comment]
let me give this a try ...

**Comment by grosser** [comment]
- [cleanup](https://github.com/grosser/ar_after_transaction/pull/41)
- [7.2 support](https://github.com/grosser/ar_after_transaction/pull/42)

**Comment by grosser** [comment]
0.12.0

**Comment by flyerhzm** [comment]
@grosser thank you.

**Pull request closed without merging**